packages/diagrams/frontend/sirius-components-diagrams/src/converter/convertBorderNodes.ts

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { GQLBorderNodePosition } from '../graphql/subscription/nodeFragment.types';
+import { BorderNodePosition } from '../renderer/DiagramRenderer.types';
+
+export const convertBorderNodePosition = (
+  initialBorderNodePosition: GQLBorderNodePosition | null,
+  defaultPosition: BorderNodePosition
```

Comment:
This parameter is useless

Reply:
Same remark. If the front does not provide a default value, then I will introduce regressions.